content/_index.md

```diff
@@ -91,20 +88,6 @@ Mehr Blogartikel lesen
 {{< /button-center >}}
 {{< /section >}}
 
-{{< section >}}
-{{< headline-icon icon="icon-calendar.svg" >}}
```

Comment:
Can you also delete the `icon-calendar.svg` file?

Reply:
done